backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/Proprietary.java

```diff
@@ -225,6 +225,21 @@ public Optional<Gegenstandlos> getGegenstandlos() {
     );
   }
 
+  /**
+   * Set the gegenstandlos metadata element (if it does not exist, it will create it).
+   * @param date - date to set as gegenstandslos with
+   */
+  public void setGegenstandlos(final String date) {
+    final Gegenstandlos gegenstandslos = getGegenstandlos()
+      .orElseGet(() -> {
+        final Element metadataParent = getOrCreateMetadataParent(Gegenstandlos.NAMESPACE);
+        return new Gegenstandlos(
+          NodeCreator.createElement(Gegenstandlos.NAMESPACE, Gegenstandlos.TAG_NAME, metadataParent)
+        );
```

Comment:
a lot of our domain classes now have a `public static Gegenstandlos createAndAppend(Node parentNode)` to keep the logic for creating themself contained within the class (not that much of a difference here, but if there are some additional properties it's helpful to have it in once place)